Jenkinsfile

```diff
@@ -68,14 +83,14 @@ stage('prep') {
 
 branches = [failFast: !fullTest]
 lines.each {line ->
-  pluginsByRepository.each { plugins ->
-    branches["pct-$plugins-$line"] = {
+  pluginsByRepository.each { repository, plugins ->
+    branches["pct-$repository-$line"] = {
```

Comment:
The cosmetic fix to the branch view: showing the repository name rather than the list of plugins in the UI.